src/Data/Expenses/Parse/Megaparsec/DateDirective.hs

```diff
@@ -46,36 +60,34 @@ dash = symbol "-"
 
 
 dayOfWeek :: Parser DT.DayOfWeek
-dayOfWeek =
-  choice
-    [ DT.Monday <$ string "MON"
-    , DT.Tuesday <$ (string "TUE" <* skipMany (noneOf "\n\r\0"))
-    , DT.Wednesday <$ (string "WED" <* skipMany (noneOf "\n\r\0"))
-    , DT.Thursday <$ (string "THU" <* skipMany (noneOf "\n\r\0"))
-    , DT.Friday <$ string "FRI"
-    , DT.Saturday <$ string "SAT"
-    , DT.Sunday <$ string "SUN"
-    ]
+dayOfWeek = choice
+  [ DT.Monday <$ string "MON"
+  , DT.Tuesday <$ (string "TUE" <* skipMany (noneOf "\n\r\0"))
+  , DT.Wednesday <$ (string "WED" <* skipMany (noneOf "\n\r\0"))
+  , DT.Thursday <$ (string "THU" <* skipMany (noneOf "\n\r\0"))
+  , DT.Friday <$ string "FRI"
+  , DT.Saturday <$ string "SAT"
+  , DT.Sunday <$ string "SUN"
+  ]
 
 
 
 date :: Parser DT.Day
-date =
-  do yyyy <- integer
-     void  dash
-     mm <- fromIntegral <$> integer
-     void  dash
-     dd <- fromIntegral <$> integer
-     return $ DT.fromGregorian yyyy mm dd
+date = do
+  yyyy <- integer
+  void dash
+  mm <- fromIntegral <$> integer
+  void dash
+  dd <- fromIntegral <$> integer
+  return $ DT.fromGregorian yyyy mm dd
 
 
 
 dateDirective :: Parser DateDirective
-dateDirective =
-  do dt <- optional date
-     case dt of
-       Just day -> do
-         dow <- option (DT.dayOfWeek day) dayOfWeek
-         return $ DateDir dt dow
-       Nothing ->
-         DateDir dt <$> dayOfWeek
+dateDirective = do
+  dt <- optional date
```

Comment:
Ah, okay. "indented left-of-thing it indents" also applies to e.g. `do` notation anyway.